<?xml version="1.0"?>
<launch>

<!-- PBVS Node -->
	<node name="pbvs" pkg="arm_vs" type="arm_vs_pbvs" output="screen">
		<param name="pbvs_control_loop_hz" value="50.0" type="double"/>
		<param name="pbvs_control_law_gain_lambda" value="1.5" type="double"/>
		<param name="pbvs_control_deadband_error" value="0.0002" type="double"/>

		<param name="desired_camera_frame" value="desired_cam_frame" type="string"/>
		<param name="current_camera_frame" value="camera_rgb_optical_frame" type="string"/>
        <param name="robot_body_frame" value="mico_link_base" type="string"/>

		<param name="control_input_topic" value="/control_input" type="string"/>
		<param name="control_input_topic_hz" value="50.0" type="double"/>
	</node>

</launch>